Libplanet/Store/LiteDBStore.cs

```diff
@@ -574,9 +571,7 @@ private string BlockStateFileId(HashDigest<SHA256> blockHash)
 
         private string StateRefId(string @namespace)
         {
-            string ns =
-                ByteUtil.Hex(namespaceHasher.ComputeHash(Encoding.UTF8.GetBytes(@namespace)));
-            return $"{StateRefIdPrefix}{ns}";
+            return $"{StateRefIdPrefix}{@namespace}";
```

Comment:
I believe this limits the possible characters used for namespaces, because LiteDB does not allow hyphens for collection names AFAIK?

On the other hand, I wish we change all `string @namespace` parameters with `Guid chainId` at all.

Reply:
According to [LiteDB's wiki][1] (it might be outdated though):

> Each collection must have a unique name:
>
> - Contains only letters, numbers and `_`
> - Collection names are **case insensitive**
> - Collection names starting with `_` are reserved for internal use
>
> The total size of all the collections names in a database is limited to 3000 bytes.

[1]: https://github.com/mbdavid/LiteDB/wiki/Collections

Reply:
I think LiteDB allows hyphen for collection names because other collection names are using namespace already. I agree to change all `string @namespace` to `Guid chainId`.